module_1/BulkFile/3_bulkFileRenaming_subdirectory_renaming.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+import shutil
+import datetime
+
+
+def movefile(datetimestr, xFile, path):
+    filename = os.path.join(path, xFile)
+    return shutil.move(filename, filename + "." + datetimestr)
+
+
+currentTime = datetime.datetime.now()
+formattedDateTime = currentTime.strftime("%Y-%m-%dT%H%M%S")
+
+path = 'temp3'
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+
+output = [movefile(formattedDateTime, f, root) for root,dirs,files in os.walk(path, topdown=False) for f in files]
```

Comment:
I'd only label the variables that are actually used here e.g. replacing `dirs` with `_`:

```python
output = [movefile(formattedDateTime, f, root) for root,_,files in os.walk(path, topdown=False) for f in files]
```